src/Storages/MergeTree/ParallelReplicasReadingCoordinator.cpp

```diff
@@ -56,6 +56,32 @@ takeFromRange(const MarkRange & range, size_t min_number_of_marks, size_t & curr
     current_marks_amount += range_we_take.getNumberOfMarks();
     return range_we_take.getNumberOfMarks();
 }
+
+void sortResponseRanges(RangesInDataPartsDescription & result)
+{
+    std::ranges::sort(result, [](const auto & lhs, const auto & rhs) { return lhs.info < rhs.info; });
```

Comment:
we have `pdqsort` in `base/base/sort.h` by calling from global namespace `::sort`
do we want to use that?

Reply:
it'd make sense for a perf critical part of the code (like sorting engine), but here I wouldn't try to be too smart